packages/sdk-router/src/constants/index.ts

```diff
@@ -1,56 +1,9 @@
 import JSBI from 'jsbi'
 import { BigNumber } from '@ethersproject/bignumber'
 
-export enum SupportedChainId {
-  ETH = 1,
-  OPTIMISM = 10,
-  CRONOS = 25,
-  BSC = 56,
-  POLYGON = 137,
-  FANTOM = 250,
-  BOBA = 288,
-  METIS = 1088,
-  MOONBEAM = 1284,
-  MOONRIVER = 1285,
-  DOGECHAIN = 2000,
-  CANTO = 7700,
-  KLAYTN = 8217,
-  ARBITRUM = 42161,
-  AVALANCHE = 43114,
-  DFK = 53935,
-  AURORA = 1313161554,
-  HARMONY = 1666600000,
-  BASE = 8453,
-}
-
-export const CCTP_ROUTER_ADDRESS: { [chainId: number]: string } = {
-  [SupportedChainId.ETH]: '0xd359bc471554504f683fbd4f6e36848612349ddf',
-  [SupportedChainId.ARBITRUM]: '0xd359bc471554504f683fbd4f6e36848612349ddf',
-  [SupportedChainId.AVALANCHE]: '0xd359bc471554504f683fbd4f6e36848612349ddf',
-  [SupportedChainId.OPTIMISM]: '0xd359bc471554504f683fbd4f6e36848612349ddf',
-}
-
-export const ROUTER_ADDRESS: { [chainId: number]: string } = {
-  [SupportedChainId.BSC]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.ETH]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.POLYGON]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.BOBA]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.MOONBEAM]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.MOONRIVER]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.ARBITRUM]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.OPTIMISM]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.AVALANCHE]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.DFK]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.FANTOM]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.HARMONY]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.AURORA]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.CRONOS]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.METIS]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.KLAYTN]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.DOGECHAIN]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.CANTO]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-  [SupportedChainId.BASE]: '0x7E7A0e201FD38d3ADAA9523Da6C109a07118C96a',
-}
+export * from './addresses'
+export * from './chainIds'
+export * from './testValues'
```

Comment:
giga-nit: exposes test values publicly to end consumers?

Reply:
@aureliusbtc  Shouldn't be so, the only the exported values from `src/index.ts` should be exposed, which do not include the constants (both directly and undirectly):

https://github.com/synapsecns/sanguine/blob/4c33414f1ac1372899fb908b387a301df4fab56c/packages/sdk-router/src/index.ts#L1-L3

Will still need to clean up `entities` and other unused stuff, but that's a story for another "purely nuking things" PR

Reply:
Well, technically the constant values **are included** in the distributed code, but given that's purely public providers, that shoudn't be an issue?

https://www.npmjs.com/package/@synapsecns/sdk-router?activeTab=code